src/core/include/units/magnitude.h

```diff
@@ -420,7 +420,8 @@ constexpr bool operator==(magnitude<LeftBPs...>, magnitude<RightBPs...>)
 template<ratio E, auto... BPs>
 constexpr auto pow(magnitude<BPs...>)
 {
-  if constexpr (E.num == 0) {
+  constexpr auto exp = E;
+  if constexpr (exp.num == 0) {
```

Comment:
Actually you should probably either leave `if constexpr (E.num == 0) {` as I fixed it or do

```cpp
  constexpr auto exp = E;
  if constexpr (exp == 0) {
```

as it was in your code.

Reply:
Reverted.  Sadly, it doesn't seem to have made a difference.